gensim/models/utils_any2vec_fast.pyx

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env cython
```

Comment:
This related only with n-grams model, I think it's better to move it to `fasttext_inner.pyx`

Reply:
I don't think this would work, since both functions are used by both `models.fasttext` and `models.keyedvectors`. Moving the functions into `models.fasttext` would break, since this would cause a circular import.

Reply:
Aha, so, in this case, better to name it as `_utils_any2vec.pyx` (similar with `_mmreader.pyx` and `_matutils.pyx`)